test/lib/testFolderDirPathConfig.json

```diff
@@ -0,0 +1,6 @@
+{"viewsPath": "mvc/viewsTest",
```

Comment:
Since this is a short list of params we should just drop these params in the constructor without needing this separate file.